### ISSUE TITLE
assigning zero to the target

### ISSUE THREAD

**Issue by shiva1393** [mravanelli/pytorch-kaldi#123]
Hi sir,
   while getting fixed shape for each batch (max_len,batch_size,inp_dim) why we are adding zero to the target(pdfid). means  zero pdfid we already have in alignments.

**Comment by TParcollet** [comment]
Hi ! Could you point to the specific part of the code ?

**Comment by shiva1393** [comment]
While passing any sequential model,
  inp= torch.zeros(max_len,batch_size,inp_dim).contiguous()
 inp[N_zeros_left:N_zeros_left+snt_len,k,:]=data_set[beg_snt:beg_snt+snt_len,:]

**Comment by mravanelli** [comment]
Hi,
in the current version, we added zeros both in the labels and in the
features. This is not an issue, because in Kaldi the label "0" is
associated to silence.

Mirco

**Comment by shiva1393** [comment]
Thank you sir.
